meteor/client/styles/tv2/keyboardPreview.scss

```diff
@@ -72,7 +68,7 @@
 	  }
 
 	  > .keyboard-preview__key__label {
-		font-size: 16px;
+		font-size: 0.59em;
 		font-weight: bold;
 		text-align: right;
 		padding: 3px;
```

Comment:
This I'm not entirely sure of. I made the labels scale consistently with the keys size, so that the keyboard has an uniform look independent from browser zoom etc., which I believe is the desired effect. But if the keys are intended to look like on the attached screenshot, I'll revert this 
![image](https://github.com/tv2/tv-automation-server-core/assets/9103996/911b5063-6af2-4338-808b-b3793e85785c)